backend/src/infra/services/shared/token/jwt-token-manager.ts

```diff
@@ -0,0 +1,40 @@
+import { CustomError } from "@/application/error";
+import { StudentRepository } from "@/application/repositories/student-repository";
+import { StudentTokenManager } from "@/application/services/token/token-manager";
+import { Student } from "@/domain";
+import jwt from 'jsonwebtoken';
+
+export class JWTStudentTokenManager implements StudentTokenManager {
+
+    SECRET_KEY: string = "unbordoJWTsecretkey*4#$@);.";
+
+    generate = async (data: object): Promise<string> => {
+        try {
+          const studentId = data.id;//getting the student id and passing on the token 
+          const token: string = jwt.sign({ _id: studentId}, this.SECRET_KEY, {
+                expiresIn: '7 days',
```

Comment:
Aqui também dá pra alterar para puxar de uma variável de ambiente, assim conseguiremos regular o tempo de expiração em desenvolvimento e produção.
```suggestion
                expiresIn: env.JWT_TOKEN_EXPIRES_IN,
```